### ISSUE TITLE
Can't use library with Swift 3

### ISSUE THREAD

**Issue by matrosovDev** [kaishin/Gifu#93]
I use this pod line in my pods file: `pod 'Gifu'`

Issues after Edit -> convert to current Swift version:

1. Method does not override any method from its superclass error.
```
override open func displayLayer(_ layer: CALayer) { 
  image = animator?.currentFrameImage ?? image
}
```

2. Contextual member 'main' has no associated value error
```
func attachDisplayLink() {
  displayLink.add(to: .main(), forMode: RunLoopMode.commonModes)
}
```

**Comment by kaishin** [comment]
Version 2.0 is not officially out yet. Please verify that you got the `2.0.0-rc` in your `Podfile.lock` and report back.

**Comment by fdiaz** [comment]
@matrosovDev maybe use:

`pod 'Gifu', :git => 'https://github.com/kaishin/Gifu.git'` in your Podfile if you want to use what's on `master` until 2.0 is officially out.

**Comment by mrachid** [comment]
Hello, i don't know why Gifu doesn't install with cocoapods, when i run pod install i have this error:

[1]    63053 abort      pod install

i don't understand, i don't have workspace

**Comment by AlexSaenko** [comment]
Hello @kaishin. When 2.0 will be officially out?

**Comment by kaishin** [comment]
@AlexSaenko This week.

**Comment by AlexSaenko** [comment]
@kaishin thank you!

**Comment by AlexSaenko** [comment]
Hello @kaishin . Happy New Year! Any news?

**Comment by kaishin** [comment]
Swift 3 support was merged in 439cc1e5c51e69c0ef6f8476dcb31df3cc5b1fa9.
